frontend/src/components/dropdown.js

```diff
@@ -135,18 +157,21 @@ export function Dropdown(props) {
       : activeItems[0].label;
   };
 
-  useOnClickOutside(contentRef, () => setDisplay(false));
-
   return (
     <div className="dib pointer relative">
-      <CustomButton onClick={toggleDropdown} className={`blue-dark ${props.className || ''}`}>
+      <CustomButton
+        ref={contentRef}
+        onClick={toggleDropdown}
+        className={`blue-dark ${props.className || ''}`}
+      >
         <div className="lh-title dib ma0 f6">{getActiveOrDisplay()}</div>
+
         <ChevronDownIcon style={{ width: '11px', height: '11px' }} className="pl3 v-mid pr1" />
       </CustomButton>
       {display && (
         <DropdownContent
+          ref={buttonRef}
```

Comment:
This is a small observation - it seems that `contentRef `and `buttonRef` have been defined the other way around. It might be more appropriate to pass `buttonRef` to the `CustomButton` component and `contentRef` to the dropdown component.

Reply:
Yes, let me make the changes